modules/local/locidex/merge/main.nf

```diff
@@ -8,7 +8,7 @@ process LOCIDEX_MERGE {
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/locidex%3A0.4.0--pyhdfd78af_0' :
         'biocontainers/locidex:0.4.0--pyhdfd78af_0' }"
-
+    containerOptions "${task.ext.containerOptions ?: ''}"
```

Comment:
If `ext.containerOptions` is configured (i.e., from the `docker` profile) it will use those options, otherwise sets options to empty string.